test(e2e): Add E2E test for Analytics creation wizard

IdrisKulubi opened the pull request:
## Closes #2033
   
   ### Changes
   - Added test IDs for Analytics creation wizard flow
   - Extended ConsolePage with createAnalytics method  
   - Created E2E test for Analytics creation via wizard
   
   ### Testing
   - E2E test passes locally
   - Follows existing test patterns 
   - Uses testWithII and initTestSuite utilities

# Motivation

-The motivation behind this PR  was to  addd the missing  E2E tests  to esure the analytics creation flow works reliably

IdrisKulubi committed:
feat: add E2E test for Analytics creation wizard

- Add test IDs for Analytics wizard components
- Extend ConsolePage with createAnalytics method
- Create E2E test following existing patterns
- Use testWithII and initTestSuite utilities

Closes #2033

peterpeterparker reviewed (commented):
Nice! 

---

1. The usage of the test IDs is incorret, can you improve that?

a. Instead of `data-testid=""` use the function `testId()`
b. Instead of harcoded string, use the constants that you declared

For example, instead of:

```
<button onclick={createOrbiter} data-testid="btn-launch-analytics">{$i18n.core.get_started}</button>
```

You should do:

```
<button onclick={createOrbiter} {...testId(testIds.createAnalytics.launch)}>{$i18n.core.get_started}</button>
```

---

2. Can you fix the various lint errors?

IdrisKulubi committed:
feat: add E2E test for Analytics creation wizard

- Add test IDs for Analytics wizard using testId() function
- Extend ConsolePage with createAnalytics method
- Create E2E test following existing satellite patterns
- Use testWithII and initTestSuite utilities

IdrisKulubi committed:
Merge branch 'main' into feat/add-analytics-e2e-test

IdrisKulubi committed:
fix(e2e): update navigation method for Analytics creation

peterpeterparker committed:
feat: revert testId usage

peterpeterparker committed:
feat: revert testId import

peterpeterparker reviewed (approved):
I'll merge the PR and fix it to finish the work.

Thx for the contribution!

Happy Hacktoberfest! 🎃🥳

peterpeterparker committed:
Merge branch 'main' into feat/add-analytics-e2e-test
